### DISCUSSION TITLE
[BUG]阿里云OSS无法上传

### DISCUSSION THREAD

**Discussion by Hideryao** [1Panel-dev/1Panel#6458] [Q&A]
### 联系方式

hiders@qq.com

### 1Panel 版本

v1.10.15-beta.1

### 问题描述

阿里云OSS账号密码都设置成功，在计划任务中进行文件夹备份时一直提醒超时错误，查看错误内容发现应该是计划任务选的网卡不对，选择的是内网网卡
报错内容
write tcp 10.0.0.105:43876->106.14.228.190:80: i/o timeout

### 重现步骤

创建OSS备份任务后就出现此问题

### 期待的正确结果

两种更正方式:
1、程序自动判断外网网卡，可进行上传任务
2、可由用户选择指定网卡进行上传任务

### 相关日志输出

```shell
write tcp 10.0.0.105:43876->106.14.228.190:80: i/o timeout
```

**Comment by wanghe-fit2cloud** [comment]
可以尝试通过修改路由表或配置特定的路由规则，来确保访问外网时使用外网网卡。

**Reply by Hideryao** [reply]
default via 10.0.0.1 dev enp0s3 proto dhcp src 10.0.0.105 metric 100 
default via 10.0.0.1 dev enp0s3 proto dhcp src 10.0.0.105 metric 1002 mtu 9000 
10.0.0.0/24 dev enp0s3 proto dhcp scope link src 10.0.0.105 metric 1002 mtu 9000 
10.0.0.1 dev enp0s3 proto dhcp scope link src 10.0.0.105 metric 100 
169.254.0.0/16 dev enp0s3 proto dhcp scope link src 10.0.0.105 metric 100 
169.254.0.0/16 dev enp0s3 proto dhcp scope link src 10.0.0.105 metric 1002 mtu 9000 
169.254.169.254 dev enp0s3 proto dhcp scope link src 10.0.0.105 metric 100 
172.17.0.0/16 dev docker0 proto kernel scope link src 172.17.0.1 linkdown 
172.18.0.0/16 dev br-2f0d2c9df12c proto kernel scope link src 172.18.0.1 linkdown 
172.19.0.0/16 dev br-a5cdc022e924 proto kernel scope link src 172.19.0.1 

打开路由表之后是这样的，求教如何设置，感谢

**Comment by Hideryao** [comment]
<img width="229" alt="image" src="https://github.com/user-attachments/assets/f5b100bf-4e92-438a-be59-c5532a03ebc7">

另外比较有意思的是，每天凌晨的时候是成功上传的，到8点以后就开始失败
